ui/app/AppLayouts/Profile/helpers/ProfileShowcaseModelAdapter.qml

```diff
@@ -65,6 +66,10 @@ QObject {
                 function getShowcaseVisibility() {
                     return Constants.ShowcaseVisibility.Everyone
                 }
+            },
+            FastExpressionRole {
+                name: "isShowcaseLoading"
+                expression: root.isShowcaseLoading
```

Comment:
`expectedRoles: []`

Reply:
Not necessary IMO since It's the default.
It would just go through this branch in the setExpressionRoles
```
    if (m_expectedRoles == expectedRoles)
        return;
```